File: src/main/java/eu/xap3y/connectfour/api/model/PlayerStatModel.java
package eu.xap3y.connectfour.api.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PlayerStatModel {

    private String uuid;
    private String name;
    private int gamesPlayed;
    private int wins;
    private int losses;
    private int draws;

}

File: core/src/main/java/eu/xap3y/connectfour/models/PlayerStatModel.kt
package eu.xap3y.connectfour.models

import kotlinx.serialization.Serializable

@Suppress("PLUGIN_IS_NOT_ENABLED")
@Serializable
data class PlayerStatModel(
    val uuid: String,
    val name: String,
    var wins: Int,
    var losses: Int,
    var draws: Int,
    var gamesPlayed: Int
)
